.github/actions/build-image-and-artifact/action.yaml

```diff
@@ -80,6 +80,11 @@ runs:
     - name: Build Core Code + Test + Create Docker Image
       shell: bash
       run: |
+        # What IG Version to use - Template used in POM for Core at this time
+        if grep -q IG_VERSION_REPLACE ${{ inputs.repositoryName }}/pom.xml; then
+          igVersion=$(sed -n 's/^IG_VERSION=\(.*\)/\1/p' < secure-api-gateway-ci/component-config/${{ inputs.repositoryName }}.env)
+          sed -i -e 's/IG_VERSION_REPLACE/'$igVersion'/g' ${{ inputs.repositoryName }}/pom.xml
```

Comment:
See comment on: https://github.com/SecureApiGateway/secure-api-gateway-core/pull/57/files#r1754385766

If you test out that command, then this can hopefully be simplified by just adding an extra arg to the mvn command used to build core.